source/installation-guide/virtual-machine.rst

```diff
@@ -40,3 +40,7 @@ Wazuh provides a pre-built virtual machine image (OVA) that you can directly imp
     # systemctl status kibana
 
 5. In order to connect to the Kibana web user interface, login with http://OVA_IP_ADRESS:5601 (where ``OVA_IP_ADDRESS`` is your system IP).
+
+.. note::
+
+        You can also download the virtual machine using the OVF format. You need to download the `OVF <https://packages.wazuh.com/vm/wazuh3.6.1_6.4.0.ovf> file and the `vmdk <https://packages.wazuh.com/vm/wazuh3.6.1_6.4.0-disk001.vmdk>` disk.
```

Comment:
Malformed links. Missing `_` character at the end of each link, producing the following output.
![error](https://user-images.githubusercontent.com/10928321/45406934-3f521c80-b668-11e8-98b2-3923731144cf.png)